docs/superdesk-ninjs-schema.json

```diff
@@ -311,6 +311,26 @@
 				}
 			}
 		},
+		"groups" : {
```

Comment:
I was wondering about the usage, maybe it would be simple if it's more like associations:
```
{
    "groups": {
        "slideshowA": {
            "items": [...],
        "slideshowB": {
            "items": [],
         }
    }
}
```
there can be only single group with given id anyhow, so the list is not needed, and this would be more consistent.
what do you think?
cc @takeit

Reply:
I think that the:

```
{
    "groups": [
        {
            "id": "slideshowA",
            "items": []
        },
        {
            "id": "slideshowB",
            "items": []
        }
    ]
}
```

should be also fine. We support it like that in the Publisher right now but your option @petrjasek also makes sense, but we would need to make some changes to support it the way you suggested now.

Reply:
I didn't realize we already use groups for packages, then it makes sense to keep it a list.
just not sure if we should combine those 2 things (packages and multi item fields)

Reply:
@petrjasek it was actually your suggestion ;) https://dev.sourcefabric.org/browse/SDESK-2955?focusedCommentId=107958&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-107958

Reply:
I know, that doesn't mean necessary it's the right thing to do :wink:

Reply:
OK so where should I put that ? It's trivial to change the name, but we need to agree on something. Something explicit like `custom_media`, or something more generic?

Reply:
I'll go with `custom_fields` if nothing better arise, and use a dict instead of list.

Reply:
@petrjasek @takeit are you fine with that?

```javascript
{"custom_fields":
    {"custom_media_id_1": [{"type": "picture", …}, …],
     "custom_media_id_2": […]
    }
}
```

Reply:
or `custom_items` to show explicitly that the values are items.

Reply:
so what will be the final format? `groups` will be renamed to `custom_fields`? because I'm lost now :)

Reply:
@jerome-poisson the `custom_items` name is fine, it's easy to change, but would that be a list at the end or not?

Reply:
@jerome-poisson I would put the items in it into `{"items": []}` to leave space for some additional metadata if needed.

Reply:
@takeit I think `groups` will stay for packages, and `custom_items` will be for multi item custom fields

Reply:
maybe we could call it `extra_items` - we put the other custom fields in `extra` so it would be kind of consistent

Reply:
@takeit @petrjasek it would be a list. So like that:
```javascript
{
    "extra_items":
    {
        "custom_media_id_1":
        {
            "items": [{"type": "picture", …}, …]
        }
        "custom_media_id_2":
        {
            "items": […]
        }
    }
}
```

Reply:
okay looks good to me 👍